WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -0,0 +1,56 @@
+package org.wordpress.android.util;
+
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import org.wordpress.android.widgets.OpenSansEditText;
+import org.wordpress.android.widgets.WPTextView;
+
+public class DialogUtils {
+    public static void showMyProfileDialog(Context context, String title, String initialText, String hint, final Callback callback) {
+        LayoutInflater layoutInflater = LayoutInflater.from(context);
+        View promptView = layoutInflater.inflate(org.wordpress.android.R.layout.my_profile_dialog, null);
```

Comment:
It's not necessary to use `org.wordpress.android.` here - it can be simply `R.layout.my_profile_dialog`. Likewise for the use places `org.wordpress.android.` is used in this class.